### PR TITLE
[O11y][Docker] Migrate Docker Overview dashboard to lens

### LINKED ISSUE #7775: [O11y][Docker] Docker Lens migration
Manually migrate `Docker` visualizations to the lens in the Kibana version `8.8.0` itself. 

## Preparation of data for testing
- [x] Use sample_event.json / setup live instance using docker / mock data using a mock server to populate dashboards 

## Migration stats
<html>
<body>

  | Before Migration |   | After Migration |  
-- | -- | -- | -- | --
  | Lens | Vizualization | Lens | Vizualization
[Metrics Docker] Overview | 0 | 7 |   |  

</body>
</html>

## Dashboard : [Metrics Docker] Overview
  - [ ] Migrate  visualization to the lens.
    - https://github.com/elastic/integrations/pull/7942
## Verification and Validation
  - [x] Verification of functionality remains the same after migration in Kibana
  - [x] Verification of data count of after-migrated visualization with before-migrated visualization in Kibana
  - [x] Verification of Dashboards is not distorted in the supported Kibana version ^8.8.0

### PR DESCRIPTION
**Pull request by rajvi-elastic**
### Urgency
- High

### Activity Type
- Enhancement

## What does this PR do?
- Manually migrate `Docker` visualizations to the lens in the current Kibana version `8.8.0` itself. 

## Migration stats
<html>
<body>

  | Before Migration |   | After Migration |  
-- | -- | -- | -- | --
  | Lens | Vizualization | Lens | Vizualization
[Metrics Docker] Overview | 0 | 7 | 7 | 0 

</body>
</html>

## Checklist

- [X] I have added an entry to my package's `changelog.yml` file.
- [X] I have verified that panels are populated with data.
- [X] I have verified that panels are not distorted after being migrated to the lens.
- [x] I have updated screenshots of the dashboard.
- [X] I have verified that the data counts are matching and panel aggregations are the same as before.

## Author's Checklist

- [X] Migrated panels should be removed from visualization folder.
- [X] Migrated visualizations are populating in current Kibana version 8.4.0 itself.

## Related issues

- Closes #7775 

## Notes
- Need to update formatting in `Docker containers` panel https://github.com/elastic/integrations/issues/7915

### CODE REVIEW

File: packages/docker/kibana/dashboard/docker-AV4REOpp5NkDleZmzKkE.json [hunk 1]
```diff
@@ -1,104 +1,504 @@
...
+                                "paging": {
+                                    "enabled": true,
+                                    "size": 8
+                                },
+                                "rowHeight": "single"
                             }
-                        }
-                    }
+                        },
+                        "title": "Docker containers [Metrics Docker] (converted)",
```
Comment (gizas) [inline packages/docker/kibana/dashboard/docker-AV4REOpp5NkDleZmzKkE.json:194]:
```suggestion
                        "title": "Docker containers [Metrics Docker]",
```

File: packages/docker/kibana/dashboard/docker-AV4REOpp5NkDleZmzKkE.json [hunk 2]
```diff
@@ -1,104 +1,504 @@
...
+                                    }
+                                ],
+                                "palette": {
+                                    "name": "default",
+                                    "type": "palette"
+                                },
+                                "shape": "donut"
+                            }
+                        },
+                        "title": "Docker containers per host [Metrics Docker] (converted)",
```
Comment (gizas) [inline packages/docker/kibana/dashboard/docker-AV4REOpp5NkDleZmzKkE.json:319]:
```suggestion
                        "title": "Docker containers per host [Metrics Docker]",
```

File: packages/docker/kibana/dashboard/docker-AV4REOpp5NkDleZmzKkE.json [hunk 3]
```diff
@@ -141,56 +880,435 @@
...
+                },
+                "gridData": {
+                    "h": 8,
+                    "i": "6cb38196-b994-40f8-b871-09d8e532a843",
+                    "w": 7,
+                    "x": 27,
+                    "y": 0
+                },
+                "panelIndex": "6cb38196-b994-40f8-b871-09d8e532a843",
+                "title": "Number of Running Containers [Metrics Docker]",
```
Comment (gizas) [inline packages/docker/kibana/dashboard/docker-AV4REOpp5NkDleZmzKkE.json:1108]:
```suggestion
                "title": "Running Containers [Metrics Docker]",
```

File: packages/docker/kibana/dashboard/docker-AV4REOpp5NkDleZmzKkE.json [hunk 4]
```diff
@@ -141,56 +880,435 @@
...
+                },
+                "gridData": {
+                    "h": 8,
+                    "i": "e5f670cb-36d3-4481-be82-733775edcb83",
+                    "w": 7,
+                    "x": 34,
+                    "y": 0
+                },
+                "panelIndex": "e5f670cb-36d3-4481-be82-733775edcb83",
+                "title": "Number of Paused Containers [Metrics Docker]",
```
Comment (gizas) [inline packages/docker/kibana/dashboard/docker-AV4REOpp5NkDleZmzKkE.json:1180]:
```suggestion
                "title": "Paused Containers [Metrics Docker]",
```

File: packages/docker/kibana/dashboard/docker-AV4REOpp5NkDleZmzKkE.json [hunk 5]
```diff
@@ -141,56 +880,435 @@
...
+                },
+                "gridData": {
+                    "h": 8,
+                    "i": "58e96448-b689-45bd-b0b2-a0ee57fc0235",
+                    "w": 7,
+                    "x": 41,
+                    "y": 0
+                },
+                "panelIndex": "58e96448-b689-45bd-b0b2-a0ee57fc0235",
+                "title": "Number of Stopped Containers [Metrics Docker]",
```
Comment (gizas) [inline packages/docker/kibana/dashboard/docker-AV4REOpp5NkDleZmzKkE.json:1252]:
```suggestion
                "title": "Stopped Containers [Metrics Docker]",
```

File: packages/docker/kibana/dashboard/docker-AV4REOpp5NkDleZmzKkE.json [hunk 6]
```diff
@@ -141,56 +880,435 @@
...
+                                    }
+                                ],
+                                "palette": {
+                                    "name": "default",
+                                    "type": "palette"
+                                },
+                                "shape": "donut"
+                            }
+                        },
+                        "title": "Docker images and names [Metrics Docker] (converted)",
```
Comment (gizas) [inline packages/docker/kibana/dashboard/docker-AV4REOpp5NkDleZmzKkE.json:1021]:
```suggestion
                        "title": "Docker images and names [Metrics Docker] ",
```

File: packages/docker/kibana/dashboard/docker-AV4REOpp5NkDleZmzKkE.json [hunk 7]
```diff
@@ -1,104 +1,504 @@
...
+                                "yLeftExtent": {
+                                    "enforce": true,
+                                    "mode": "full"
+                                },
+                                "yLeftScale": "linear",
+                                "yRightScale": "linear",
+                                "yTitle": "Count"
+                            }
+                        },
+                        "title": "CPU usage [Metrics Docker] (converted)",
```
Comment (gizas) [inline packages/docker/kibana/dashboard/docker-AV4REOpp5NkDleZmzKkE.json:496]:
```suggestion
                        "title": "CPU usage [Metrics Docker]",
```

File: packages/docker/kibana/dashboard/docker-AV4REOpp5NkDleZmzKkE.json [hunk 8]
```diff
@@ -108,15 +508,174 @@
...
+                                "yLeftExtent": {
+                                    "enforce": true,
+                                    "mode": "full"
+                                },
+                                "yLeftScale": "linear",
+                                "yRightScale": "linear",
+                                "yTitle": "Count"
+                            }
+                        },
+                        "title": "Memory usage [Metrics Docker] (converted)",
```
Comment (gizas) [inline packages/docker/kibana/dashboard/docker-AV4REOpp5NkDleZmzKkE.json:673]:
```suggestion
                        "title": "Memory usage [Metrics Docker]",
```

File: packages/docker/kibana/dashboard/docker-AV4REOpp5NkDleZmzKkE.json [hunk 9]
```diff
@@ -126,12 +685,192 @@
...
+                                "yLeftExtent": {
+                                    "enforce": true,
+                                    "mode": "full"
+                                },
+                                "yLeftScale": "linear",
+                                "yRightScale": "linear",
+                                "yTitle": "Count"
+                            }
+                        },
+                        "title": "Network IO [Metrics Docker] (converted)",
```
Comment (gizas) [inline packages/docker/kibana/dashboard/docker-AV4REOpp5NkDleZmzKkE.json:868]:
```suggestion
                        "title": "Network IO [Metrics Docker] ",
```

File: packages/docker/kibana/dashboard/docker-AV4REOpp5NkDleZmzKkE.json [hunk 10]
```diff
@@ -1,104 +1,504 @@
...
-                    "h": 8,
-                    "i": "2",
-                    "w": 20,
-                    "x": 28,
-                    "y": 0
-                },
-                "panelIndex": "2",
-                "panelRefName": "panel_1",
-                "version": "7.3.0"
+                "title": "Docker containers [Metrics Docker]",
```
Comment (gizas) [inline packages/docker/kibana/dashboard/docker-AV4REOpp5NkDleZmzKkE.json:209]: Do we need [Metrics Docker] to be visible to all titles? Shall we remove it?

Comment (rajvi-elastic) [reply packages/docker/kibana/dashboard/docker-AV4REOpp5NkDleZmzKkE.json:209]: @gizas, According to Elastic [documentation](https://www.elastic.co/guide/en/integrations-developer/current/create-dashboards.html#_visualizations) for integration development it is recommended to add suffix `<NAME> [<Metrics | Logs> <PACKAGE NAME>]`

Comment (constanca-m) [reply packages/docker/kibana/dashboard/docker-AV4REOpp5NkDleZmzKkE.json:209]: I think it is best to remove it, according to the [dashboard guidelines](https://github.com/elastic/integrations/blob/main/docs/dashboard_guidelines.md#titles-in-visualisations-matter).

Comment (rajvi-elastic) [reply packages/docker/kibana/dashboard/docker-AV4REOpp5NkDleZmzKkE.json:209]: Here is the [issue](https://github.com/elastic/integrations/issues/7999) for removing suffix from docker visualization.

**Review by constanca-m — APPROVED** [review]

**Review by milan-elastic — APPROVED** [review]
LGTM!

**Pull request merged**